zokrates_stdlib/stdlib/hashes/sha256/embed/512bit.zok

```diff
@@ -3,13 +3,8 @@ import "./shaRoundNoBoolCheck" as sha256
 
 // A function that takes 2 bool[256] arrays as inputs
 // and returns their sha256 compression function as an array of 256 bool.
-// In contrast to full_round.zok no padding is being applied
+// No padding is being applied
 def main(bool[256] a, bool[256] b) -> (bool[256]):
-
-    // a and b is NOT checked to be of type bool
```

Comment:
these comments are outdated? @Schaeff

Reply:
Good catch, yes this is before we had bool